components/mdns/mdns.c

```diff
@@ -26,8 +26,7 @@
 #endif
 
 #if ESP_IDF_VERSION <= ESP_IDF_VERSION_VAL(5, 1, 0)
-// IDF <= v5.1 does not support enabling/disabling esp-wifi
-#define MDNS_ESP_WIFI_ENABLED 1
+#define MDNS_ESP_WIFI_ENABLED CONFIG_SOC_WIFI_SUPPORTED
```

Comment:
Is this `CONFIG_SOC_WIFI_SUPPORTED` defined in say `v4.3` ?

Reply:
`mdns` as a component should be used only if `idf: ">=5.0"`, why we need to check `v4.3` ?

Reply:
ah so true... mdns is part of IDF before 5.0...

Reply:
I see. no more concerns if we support mdns only from `v5.0`